sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ComplexTypeSuite.scala

```diff
@@ -118,19 +118,20 @@ class ComplexTypeSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("CreateArray") {
-    val intSeq = Seq(5, 10, 15, 20, 25)
-    val longSeq = intSeq.map(_.toLong)
-    val strSeq = intSeq.map(_.toString)
-    checkEvaluation(CreateArray(intSeq.map(Literal(_))), intSeq, EmptyRow)
-    checkEvaluation(CreateArray(longSeq.map(Literal(_))), longSeq, EmptyRow)
-    checkEvaluation(CreateArray(strSeq.map(Literal(_))), strSeq, EmptyRow)
-
-    val intWithNull = intSeq.map(Literal(_)) :+ Literal.create(null, IntegerType)
-    val longWithNull = longSeq.map(Literal(_)) :+ Literal.create(null, LongType)
-    val strWithNull = strSeq.map(Literal(_)) :+ Literal.create(null, StringType)
-    checkEvaluation(CreateArray(intWithNull), intSeq :+ null, EmptyRow)
-    checkEvaluation(CreateArray(longWithNull), longSeq :+ null, EmptyRow)
-    checkEvaluation(CreateArray(strWithNull), strSeq :+ null, EmptyRow)
+    // Array is required to pass Array(_, containsNull = false) as type information
```

Comment:
Is this comment correct? `containsNull` in `CreateArray` is determined by `containsNull = children.exists(_.nullable)`. Does it matter it is `Array` or not?